deny.toml

```diff
@@ -73,6 +73,7 @@ ignore = [
   { id = "RUSTSEC-2020-0168", reason = "There is no suitable replacement for mach and the mach2 crate has not been vetted." },
   { id = "RUSTSEC-2024-0336", reason = "Only use of rustls@v0.20.9 is in futures-rustls which does not use the effected code" },
   { id = "RUSTSEC-2024-0344", reason = "We are only able to remove this once parity updates its dependencies. Older versions of curve25519-dalek should get replaces with >= 4.1.3" },
+  { id = "RUSTSEC-2022-0093", reason = "Pending litep2p upgrade. Vulnerable code not exploitable in Frequency." },
```

Comment:
I think it should say why it's not exploitable here.  This is about edd25519-dalek, which we don't use, right?